pkg/router/template/router.go

```diff
@@ -1489,3 +1492,24 @@ func privateKeysFromPEM(pemCerts []byte) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+// watchCABundleCert watches the directory containing the CA bundle certificate
+// and reloads the router if the directory contents change.
+func (r *templateRouter) watchCABundleCert() error {
+	caBundleDir := filepath.Dir(r.defaultDestinationCAPath)
+
+	reloadFn := func() {
+		log.V(0).Info("router slated for reload after detecting changes in CA bundle certificate directory")
```

Comment:
This is a bit more verbose than the log message for reloading the default certificate ("reloading to get updated default certificate") or client CA CRL ("reloading to get updated client CA CRL").  Is there a reason not to keep it simple (e.g., "reloading to get updated default destination CA certificate bundle")?

I do like that the client CA CRL reload function also logs the file name; that might be useful here too.
https://github.com/openshift/router/blob/b59b5f605294c19b27c6c5527718bc3fca431cc0/pkg/router/template/router.go#L470